src/validate.js

```diff
@@ -30,24 +29,45 @@ var resolutionSchema = exports.resolutionSchema = Joi.alternatives().try(
 
 var browsersSchema = exports.browsersSchema = Joi.array().min(1).unique().items(
   Joi.object().keys({
-    browserName: Joi.string().valid(['chrome', 'firefox']).required(),
+    browserName: Joi.string().valid('chrome', 'firefox').required(),
     includeRules: includeRulesSchema,
     excludeRules: excludeRulesSchema
   }),
   Joi.object().keys({
-    browserName: Joi.string().valid(['chrome', 'firefox', 'safari', 'microsoftedge', 'internet explorer']).required(),
-    version: Joi.string().required(),
+    browserName: Joi.string().valid('internet explorer').required(),
+    version: Joi.string().regex(/^[1-9]\d+$/).required(),
```

Comment:
@xwa130saucer this should only support `11`

Reply:
addressed